Admin-app/src/components/ChatList/ChatList.js

```diff
@@ -72,9 +72,9 @@ export default function ChatList(props) {
 
     // This hook shows all the clients that belong to the doctor as a list. 
     useEffect(() => {
-        onSnapshot(q, (doc) => {
-            setClients(doc.docs.map(doc => ({
-                name: doc.data().name,
+        onSnapshot(adminRef, (doc) => {
+            setClients(doc.data().treats?.map(patient => ({
+                name: patient
```

Comment:
I've looked over your code, and it looks good. 

My only question is within ChatList, you are performing setClients(...) which I'm assuming is supposed to store all the clients that belong to the doctor.

However, I did console.log(clients) and instead of returning the 3 patients assigned, it only returned 1. Is this normal? If not, I think you might be overriding instead of appending.

Reply:
So, when I was testing it out, I was adding and removing doctors, if you go to inbox, you should see a list of all doctors. If you dont let me know, With my testing, I added, and removed patients from my doctors profile and was able to see the changes.